config/admission/admission.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: "clusterextensions-package-uniqueness.olm.operatorframework.io"
+spec:
+  failurePolicy: Fail
+  paramKind:
+    apiVersion: olm.operatorframework.io/v1alpha1
+    kind: ClusterExtension
+  matchConstraints:
+    resourceRules:
+    - apiGroups:   ["olm.operatorframework.io"]
+      apiVersions: ["v1alpha1"]
+      operations:  ["CREATE", "UPDATE"]
+      resources:   ["clusterextensions"]
+  validations:
+    - expression: "object.spec.packageName != params.spec.packageName"
+      messageExpression: "'Package \"' + string(object.spec.packageName) + '\" is already installed via ClusterExtension \"' +  string(params.metadata.name) + '\"'"
+      reason: Invalid
+
+---
+
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingAdmissionPolicyBinding
+metadata:
+  name: "clusterextensions-package-uniqueness-binding.olm.operatorframework.io"
```

Comment:
After kustomize it ends up being `operator-controller-clusterextensions-package-uniqueness-binding.olm.operatorframework.io`. I think I'll drop `.olm.operatorframework.io` part to make it a bit shorter.

Same with `ValidatingAdmissionPolicy`